src/main/Config.h

```diff
@@ -190,9 +194,10 @@ class Config : public std::enable_shared_from_this<Config>
     bool ARTIFICIALLY_REPLAY_WITH_NEWEST_BUCKET_LOGIC_FOR_TESTING;
 
     // A config parameter that forces transaction application during ledger
-    // close to sleep for a given number of microseconds. This option is only
-    // for consensus and overlay simulation testing.
-    uint32_t OP_APPLY_SLEEP_TIME_FOR_TESTING;
+    // close to sleep. This option is only for consensus and overlay simulation
+    // testing. This vector contains 100 elements each of which is a duration.
+    // Some durations repeat themselves to achieve a desired distribution.
+    std::vector<std::chrono::microseconds> OP_APPLY_SLEEP_TIME_FOR_TESTING;
```

Comment:
correction: the member var should probably live in the Config class, as it's not really LedgerManager's responsibility to maintain this config.

Reply:
After discussing this offline & some more thinking, I updated this such that:

- `processOpApplySleepTimeForTestingConfigs` is a public method that takes the weight and duration lists and produces a vector of 100 elements. (I made it public so I can run it from `CommandLine::run`)
- `opApplySleepTimeForTesting` is a public member that contains 100 elements. (I made it public and removed the `choose` function because `rand_element` makes it simple and also it can be used to check if we need to simulate op apply.)

Let me know what you think!